src/main/java/com/exasol/adapter/dynamodb/documentpath/LinearDocumentPathWalker.java

```diff
@@ -0,0 +1,48 @@
+package com.exasol.adapter.dynamodb.documentpath;
+
+import com.exasol.adapter.dynamodb.documentnode.DocumentNode;
+
+/**
+ * This class is a simplified version of {@link DocumentPathWalker}. It has the limitation that it can only walk linear
+ * paths. That means that the paths must not contain {@link ArrayAllPathSegment}s.
+ *
+ * In contrast to {@link DocumentPathWalker#walk(DocumentNode)} This classes walk method only returns one single value.
```

Comment:
```suggestion
 * In contrast to {@link DocumentPathWalker#walk(DocumentNode)}, this class's walk method returns one single value.
```

Reply:
`only` and `one` were saying same thing, I removed only. You can also change it to `only returns a single value.`.